tests/pom.xml

```diff
@@ -120,21 +125,15 @@
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
-            <version>${slf4j-api.version}</version>
-            <scope>provided</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.logging.log4j</groupId>
-            <artifactId>log4j-core</artifactId>
-            <version>${slf4j-log4j.version}</version>
-            <scope>test</scope>
+            <scope>compile</scope>
```

Comment:
Why the change from **provided** to **compile** scope for this?

Reply:
my bad, there is no need for that